.github/workflows/projects.yml

```diff
@@ -0,0 +1,168 @@
+        run: |
+          ./vsdownload.py --accept-license --dest $(pwd)/msvc
+          ./install.sh $(pwd)/msvc
+      - name: Checkout gas-preprocessor
+        uses: actions/checkout@v3
+        with:
+          repository: ffmpeg/gas-preprocessor
+          ref: 9309c67acb535ca6248f092e96131d8eb07eefc1
+          path: gas-preprocessor
+      - name: Add gas-preprocessor to path
+        run: |
+          echo $(pwd)/gas-preprocessor >> $GITHUB_PATH
+      - name: Checkout dav1d
+        run: |
+          git clone https://code.videolan.org/videolan/dav1d.git
+          cd dav1d
+          git checkout 1.2.1
+      - name: Build dav1d
+        run: |
+          cd dav1d
+          mkdir build
+          cd build
+          export PATH=$(pwd)/../../msvc/bin/arm64:$PATH
+          cat >cross.txt <<EOF
+          [binaries]
+          c = 'cl'
+          cpp = 'cl'
+          ar = 'lib'
+          windres = 'rc'
+          ;exe_wrapper = ['wine']
+
+          [properties]
+          needs_exe_wrapper = true
+
+          [host_machine]
+          system = 'windows'
+          cpu_family = 'aarch64'
+          cpu = 'aarch64'
+          endian = 'little'
+          EOF
+
+          meson setup --cross-file cross.txt --buildtype release --prefix $(pwd)/../../install ..
+          ninja -v
+          ninja install
+          cp ../../msvc/vc/Redist/MSVC/1*/arm64/*.CRT/vcruntime*.dll ../../install/bin
+      # The build output from MSVC actually is redistributable, even if the
+      # toolchain itself isn't.
+      - uses: actions/upload-artifact@v3
+        with:
+          name: dav1d
+          path: |
+            install/*
+          retention-days: 7
+
+  test-build-ffmpeg:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install prerequisites
+        run: |
+          sudo apt-get update && sudo apt-get install wine64-development python3 msitools python3-simplejson python3-six ca-certificates winbind nasm
+          wine64 wineboot
+      - uses: actions/checkout@v3
+      - name: Download MSVC
+        run: |
+          ./vsdownload.py --accept-license --dest $(pwd)/msvc
+          ./install.sh $(pwd)/msvc
+      - name: Checkout gas-preprocessor
+        uses: actions/checkout@v3
+        with:
+          repository: ffmpeg/gas-preprocessor
+          ref: 9309c67acb535ca6248f092e96131d8eb07eefc1
+          path: gas-preprocessor
+      - name: Add gas-preprocessor to path
+        run: |
+          echo $(pwd)/gas-preprocessor >> $GITHUB_PATH
+      - name: Checkout ffmpeg
+        uses: actions/checkout@v3
+        with:
+          repository: ffmpeg/ffmpeg
+          ref: n6.0
+          path: ffmpeg
+      - name: Build ffmpeg
+        run: |
+          mkdir ffmpeg-build
+          cd ffmpeg-build
+          export PATH=$(pwd)/../msvc/bin/arm64:$PATH
+          ../ffmpeg/configure --arch=arm64 --target-os=win32 --toolchain=msvc --enable-cross-compile --prefix=$(pwd)/../install
+          make -j$(nproc)
+          make install
```

Comment:
Unlike other two projects, you don't copy vc runtime dll here, is it because ffmpeg links to static runtime?

Reply:
Yes, that's correct - by default, ffmpeg doesn't add any `-MD` options or similar, so it defaults to what `cl.exe` defaults to, which is static linking of the runtime.

Reply:
ok.